content/applications/finance/accounting/customer_invoices.rst

```diff
@@ -4,225 +4,221 @@
+- To access the list of customers requiring action along with the :guilabel:`Total Due` and
+  :guilabel:`Total Overdue` amounts, go to
+  :menuselection:`Accounting --> Customers --> Follow-up Reports`.
+- To get a full overview of the customer's open invoices, click on its line.
+- To open a payment reminder to be sent to the customer, click the :guilabel:`Follow up` button.
+- To exclude any invoices from the follow-up, enable the option :guilabel:`Exclude from Follow-ups`
+  for the selected ones.
 
-Apply the payment. Below, you can see that the payment has been added to
-the invoice.
+.. _accounting/invoices/aging-report:
 
-.. image:: customer_invoices/invoice10.png
+Aged Receivable
+---------------
 
-Payment Followup
-----------------
+To review outstanding customer invoices and their related due dates, use the
+:ref:`Aged Receivable <accounting/reporting/aged-receivable>` report. To access it, go to
+:menuselection:`Accounting --> Reporting --> Aged Receivable`.
 
-There's a growing trend of customers paying bills later and later.
-Therefore, collectors must make every effort to collect money and
-collect it faster.
+.. _accounting/invoice/reporting:
 
-Odoo will help you define your follow-up strategy. To remind customers
-to pay their outstanding invoices, you can define different actions
-depending on how severely overdue the customer is. These actions are
-bundled into follow-up levels that are triggered when the due date of an
-invoice has passed a certain number of days. If there are other overdue
-invoices for the same customer, the actions of the most overdue invoice
-will be executed.
+Reporting
+=========
 
-By going to the customer record and diving into the "Overdue Payments"
-you will see the follow-up message and all overdue invoices.
+Profit and Loss
+---------------
 
-.. image:: customer_invoices/invoice11.png
+The :ref:`Profit and Loss <accounting/reporting/profit-and-loss>` statement shows details of income
+and expenses.
 
-.. image:: customer_invoices/invoice12.png
+Balance sheet
+-------------
 
-.. _customer-invoices/aging-report:
+The :ref:`Balance Sheet <accounting/reporting/balance-sheet>` summarizes the company's liabilities,
+assets, and equity at a specific time.
 
-Customer aging report:
-~~~~~~~~~~~~~~~~~~~~~~
+.. _accounting/invoice/resequencing:
 
-The customer aging report will be an additional key tool for the
-collector to understand the customer credit issues, and to prioritize
-their work.
+Resequencing
+============
 
-Use the aging report to determine which customers are overdue and begin
-your collection efforts.
+:guilabel:`Resequence` is a technical feature used to change the sequence of documents. It can be
+helpful to resequence invoice numbers when importing invoices from another invoicing or accounting
+system, mainly if the reference originates from the previous software and continuity for the current
+year must be maintained without restarting from the beginning.
```

Comment:
It is also possible to change the sequence of invoices to include information such as the month and year. I actually learned something new from this forum post! https://www.odoo.com/forum/help-1/how-to-resequence-invoices-in-odoo-with-month-and-year-257878#:~:text=If%20we%20want%20to%20add,term%20as%20year%20and%20month.
I'm not sure if we want to include this as another method of resequencing or try to separate it a bit. Let's discuss when we're both back :)

Reply:
Maybe this section should be called Sequences and then we can have a subsection that is "Resequencing". The first would explain what they are, how to set them as explained by that forum post, etc. The subsection would explain how to resequence them as you have explained.

Reply:
Should I take the paragraph from the 'Invoice confirmation' part for the new 'Sequence' section? I think what the forum post explains should be in the 'Resequencing' section as they change the sequence.

Reply:
I was thinking of how new users might want to set their invoices up like this from the beginning, but you're right that it is technically resequencing. Let's call what you've described "mass resequencing" and the method of the forum post "resequencing". This will bring a few changes:
- Mass resequencing is only available to users with administrator or advisor Accounting access (not the case for individual resequencing).
- Neither requires developer mode as of my testing; let me know if you find otherwise.